src/main/resources/changelog/db-changelog-master.xml

```diff
@@ -0,0 +1,41 @@
+<databaseChangeLog
```

Comment:
This changelog file clashes with the other changelog file. Merge them into one config file and remove this.